tck/cmd/server.go

```diff
@@ -29,16 +29,10 @@ func main() {
 
 	// Create a new RPC server
 	assigner := handler.Map{
-		"setup":                  errorHandlerWrapper(HandleError, handler.New(sdkService.Setup)),
-		"reset":                  errorHandlerWrapper(HandleError, handler.New(sdkService.Reset)),
-		"createAccount":          errorHandlerWrapper(HandleError, handler.New(accountService.CreateAccount)),
-		"createAccountFromAlias": errorHandlerWrapper(HandleError, handler.New(accountService.CreateAccountFromAlias)),
-		"getAccountInfo":         errorHandlerWrapper(HandleError, handler.New(accountService.GetAccountInfo)),
-		"updateAccountMemo":      errorHandlerWrapper(HandleError, handler.New(accountService.UpdateAccountMemo)),
-		"updateAccountKey":       errorHandlerWrapper(HandleError, handler.New(accountService.UpdateAccountKey)),
-		"deleteAccount":          errorHandlerWrapper(HandleError, handler.New(accountService.DeleteAccount)),
-		"generatePublicKey":      errorHandlerWrapper(HandleError, handler.New(methods.GeneratePublicKey)),
-		"generatePrivateKey":     errorHandlerWrapper(HandleError, handler.New(methods.GeneratePrivateKey)),
+		"setup":         errorHandlerWrapper(HandleError, handler.New(sdkService.Setup)),
```

Comment:
We are binding the endpoints to `errorHandlerWrapper`. This doesn't look that good in my opinion. Would expect something like `setup: handler.New()`). Maybe we can rename to `HandleWrapper`.

Reply:
Renamed this to `postHandler`. Have a look.